services/httpd/handler.go

```diff
@@ -1076,22 +1076,21 @@ func (h *Handler) servePromRead(w http.ResponseWriter, r *http.Request, user met
 		atomic.AddInt64(&h.stats.QueryRequestBytesTransmitted, int64(len(compressed)))
 	}
 
+	resp := &remote.ReadResponse{
```

Comment:
I don't see `resp` being used before line 1090. Why did you need to move the `resp := ` assignment up above the `Store.Read` block?

Reply:
Just felt better there to me but I moved it back to where it was before